lavamoat/browserify/main/policy.json

```diff
@@ -1697,7 +1697,7 @@
         "@metamask/json-rpc-engine": true,
         "@metamask/rpc-errors": true,
         "@metamask/selected-network-controller": true,
-        "@metamask/utils": true
+        "@metamask/queued-request-controller>@metamask/utils": true
```

Comment:
This changed because the extension and this package now use different versions of `@metamask/utils`.